docker/requirements.txt

```diff
@@ -53,13 +53,13 @@ urbanaccess
 
 # jupyter things
 jupyter==1.0.0
-jupyterlab==1.0.9
+jupyterlab==1.2.3
 jupyterhub==0.9.6
```

Comment:
can we increase version of jupyterhub too? the hub is using 1.1.0 and I believe they should match?

Reply:
`jupyterlab==1.2.3` was released Nov 13 2019, `jupyterhub==1.1.0` was released jan 17, 2020.

I will test this first.